straitlets/builtin_models.py

```diff
@@ -14,9 +15,9 @@ def join_filter_empty(sep, *elems):
 
     Example
     -------
-    >>> join_if_second_nonempty(':', 'a', None, 'c')
+    >>> join_filter_empty(':', 'a', None, 'c')
     'a:c'
-    >>> join_if_second_nonempty(':', 'a', None)
+    >>> join_filter_empty(':', 'a', None)
     'a'
     """
     return sep.join(map(str, filter(bool, elems)))
```

Comment:
I learned recently that `filter(bool, <sequence>)` is actually equivalent to `filter(None, <sequence>)`, except there's a fast path in the implementation of `filter` when `None` is passed.

Probably doesn't matter here for performance, but if I were rewriting this function, I'd probably pass `None` here.